distribution/pom.xml

```diff
@@ -83,6 +74,7 @@
 				<executions>
 					<execution>
 						<id>attach-javadoc</id>
+						<phase>install</phase>
```

Comment:
If these plugins are painful (slowness is the only reason I could think of for delaying their creation), we could create a specific profile for them, rather than assembling them in the same phase that they are being pushed to the local repository in. The order of plugins in the install phase would then determine exactly how the artifacts were pushed, which doesn't seem like it would be a good thing in the long term if builds are not repeatable.